roles/user/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+# Create EDA Controller user
+- name: Add EDA Controller user
+  user:
```

Comment:
We don't tend to because we may end up having two different namespaces. I've commited a change to ignore that on ansible-lint. 

It's throwing a false positive here because the `user` module is a core module which there are extra ansible-lint rules about.